lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -357,7 +357,7 @@ defmodule Kernel.SpecialForms do
       iex> Kernel.'+'(1, 2)
       3
 
-  Note that `Kernel."HELLO"` will be treated as a remote call and not an alias.
+  Note that `Kernel."function_name"` will be treated as a remote call and not an alias.
```

Comment:
I see. but.. how can a function be named Hello or HELLO since functions must start with lowercase or underscore? or is there any other remote call that will take this format?